llvm/test/Transforms/LoopVectorize/AArch64/sve-vfabi.ll

```diff
@@ -0,0 +1,110 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
+; RUN: opt < %s -passes=loop-vectorize,simplifycfg,instcombine -force-vector-interleave=1 -prefer-predicate-over-epilogue=predicate-dont-vectorize -S | FileCheck %s
+
+target triple = "aarch64-unknown-linux-gnu"
+
+define void @test_big_little_params(ptr readonly %a, ptr readonly %b, ptr noalias %c) #0 {
```

Comment:
I think that would be better to add more unit tests to the VectorFunctionABITests.cpp to cover mixed scalar types, as that ensures that you are testing your actual changes in the code, the LV tests rely on too many things in my opinion.